bedrock/base/templates/macros-protocol.html

```diff
@@ -250,6 +250,7 @@
         alt=image_alt,
         class='mzp-c-card-image',
         height=image_height,
+        highres_image_url=highres_image_url,
         include_highres=include_highres_image,
```

Comment:
The root of this issue might be in our ContentCardManager model for get_data
<img width="752" alt="Screen Shot 2022-10-03 at 4 03 52 PM" src="https://user-images.githubusercontent.com/19650432/193613115-644cc2d0-1379-4794-bd6b-e3ad1d0ff0b9.png">